### PR TITLE
SW-5520: Implemented redux for planting site create, validate, update, delete

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

File: src/api/types/generated-schema.ts [hunk 1]
```diff
@@ -3822,7 +3838,7 @@ export interface components {
       rare?: boolean;
       scientificName: string;
       /** @enum {string} */
-      seedStorageBehavior?: "Orthodox" | "Recalcitrant" | "Intermediate" | "Unknown" | "Likely Orthodox" | "Likely Recalcitrant" | "Likely Intermediate";
+      seedStorageBehavior?: "Orthodox" | "Recalcitrant" | "Intermediate" | "Unknown" | "Likely Orthodox" | "Likely Recalcitrant" | "Likely Intermediate" | "Intermediate - Cool Temperature Sensitive" | "Intermediate - Partial Desiccation Tolerant" | "Intermediate - Short Lived";
```
Comment (nickgraz) [inline src/api/types/generated-schema.ts:3841]: I believe these changes were already made by Steve, can you sync this branch with main?

File: src/redux/rootReducer.ts [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-import projectsReducers from 'src/redux/features/projects/projectsSlice';
-import reportsSettingsReducers from 'src/redux/features/reportsSettings/reportsSettingsSlice';
-import speciesReducers from 'src/redux/features/species';
-
 import acceleratorReducers from './features/accelerator/acceleratorSlice';
 import accessionsReducers from './features/accessions/accessionsSlice';
 import appVersionReducers from './features/appVersion/appVersionSlice';
 import batchesReducers from './features/batches/batchesSlice';
+import cohortsReducers from './features/cohorts/cohortsSlice';
+import deliverablesReducers from './features/deliverables/deliverablesSlice';
```
Comment (nickgraz) [inline src/redux/rootReducer.ts:8]: <3

**Review by nickgraz — APPROVED** [review]
Great stuff. Are you going to do those TODOs in your current work or make a ticket for later?

**Comment by tommylau523** [comment]
### Merge activity

* **Jun 11, 8:54 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2720).

**Pull request merged**